applications/nyaml/NXarpes.yaml

```diff
@@ -3,6 +3,10 @@ doc: |
   This is an application definition for angular resolved photo electron spectroscopy.
   
   It has been drawn up with hemispherical electron analysers in mind.
+
+  This definition is a legacy support for older NXarpes experiments.
+  There is, however, a newer definition to collect data & metadata
+  for general photoemission experiments, called :ref:`NXmpes`.
```

Comment:
I would like to come back to this PR. I think here, there should be a reference to the now-merged `NXmpes_arpes`, but then we can merge it.

Reply:
Updated it

Reply:
Thanks. Sorry, I was a bit unclear what I meant. I was thinking something like

"There is, however, a newer definition to collect data & metadata for general photoemission experiments, called :ref:`NXmpes`, as well as a specialization for ARPES experiments, called :ref:`NXmpes_arpes`."

What do you think?